opentracing/tracer.py

```diff
@@ -40,31 +41,84 @@ class Tracer(object):
     def __init__(self):
         self._noop_span_context = SpanContext()
         self._noop_span = Span(tracer=self, context=self._noop_span_context)
+        self._active_span_source = NoopActiveSpanSource()
 
-    def start_span(self,
-                   operation_name=None,
-                   child_of=None,
-                   references=None,
-                   tags=None,
-                   start_time=None):
+    @property
+    def active_span_source(self):
```

Comment:
Sure - when users call `start_manual_span` and later may need to access the `ActiveSpanSource` to call `make_active` themselves.